packages/BetterPhpDocParser/PhpDocInfo/PhpDocInfo.php

```diff
@@ -254,11 +250,14 @@ public function findOneByAnnotationClass(string $desiredClass): ?DoctrineAnnotat
      * @template T of \PHPStan\PhpDocParser\Ast\Node
      * @param class-string<T> $typeToRemove
      */
-    public function removeByType(string $typeToRemove): void
+    public function removeByType(string $typeToRemove): bool
```

Comment:
The docblock changes should not be flying around 3 services, but return true/false in case of change/no change.
Then Rector class should then pick the bool value and return node for change, or `null` for no change.